src/main/resources/application.yml

```diff
@@ -0,0 +1,8 @@
+spring:
+    datasource:
+        driverClassName: com.mysql.jdbc.Driver
+        url: jdbc:mysql://localhost:3306/chipdale
+        username: hejcz
+        password: kalafior
```

Comment:
Myślę, że ujdzie. Można wpisać username: username itd. Nie ma to chyba znaczenia skoro to dane do localhosta. !resolved